src/lib/utils/i18n/IInternationalization.ts

```diff
@@ -0,0 +1,47 @@
+import type { Message } from 'discord.js';
+import type { Awaited } from '../Types';
+
+export interface IInternationalization {
+	/**
+	 * Resolves an i18n key from a message.
+	 * @param message The message for context.
+	 * @example
+	 * ```typescript
+	 * // Example usage:
```

Comment:
Literally can't click on the line below, shouldn't it be... `const language` or code or similar? Same with its function

Reply:
Technically it's the language's name, the method is called `resolveNameFromMessage` after all.